### ISSUE TITLE
Raw docstring not ignored when computing similarities

### ISSUE THREAD

**Issue by kimt33** [PyCQA/pylint#2829]
Raw docstrings are not ignored with the `ignore-docstring=yes` in the similarities checker. I think it should be ignored because raw docstrings are also considered a docstring according to [PEP 257](https://www.python.org/dev/peps/pep-0257/)
### Steps to reproduce
Run pylint on the following two files:
`pylint ./one.py ./two.py`
```python
"""One module"""


def one_func():
    r"""hello

    This docstring is duplicated.
    This docstring is duplicated.
    This docstring is duplicated.
    This docstring is duplicated.
    """
    return 1
```
```python
"""Another module"""


def another_func():
    r"""hello

    This docstring is duplicated.
    This docstring is duplicated.
    This docstring is duplicated.
    This docstring is duplicated.
    """
```

### Current behavior
Pylint reports:
```
************* Module temp.pylint_test.two
two.py:1:0: R0801: Similar lines in 2 files
==temp.pylint_test.one:4
==temp.pylint_test.two:4
    r"""hello

    This docstring is duplicated.
    This docstring is duplicated.
    This docstring is duplicated.
    This docstring is duplicated.
    """ (duplicate-code)

------------------------------------------------------------------
Your code has been rated at 6.67/10 (previous run: 6.67/10, +0.00)
```

### Expected behavior
Since the docstrings are ignored by default, raw docstrings should also be ignored: 
```
--------------------------------------------------------------------
Your code has been rated at 10.00/10 (previous run: 10.00/10, +0.00)
```
### pylint --version output
```
pylint 2.3.1
astroid 2.2.5
Python 3.6.8 |Anaconda, Inc.| (default, Dec 30 2018, 01:22:34) 
[GCC 7.3.0]
```

**Comment by PCManticore** [comment]
Thanks @kimt33 This makes sense.
